imapsync/usr/lib/systemd/system/imapsync@.service

```diff
@@ -12,5 +12,5 @@ ExecStart=/usr/bin/imapsync --noauthmd5 --noreleasecheck --allowsizemismatch \
 --passfile2 /var/lib/nethserver/imapsync/vmail.pwd \
 --host1 ${REMOTEHOSTNAME} --port1 ${REMOTEPORT} ${SECURITY} --user1 ${REMOTEUSERNAME} \
 --passfile1 /var/lib/nethserver/imapsync/%i.pwd \
---exclude '^Public Folders$|^Trash$|^Deleted Items$|^Public${EXCLUDE}' \
+--exclude '^Public Folders|^Trash|^Deleted Items|^Public${EXCLUDE}' \
```

Comment:
Systemd command lines are really tricky as they resemble Bash, but they are not Bash at all. Are you sure the value of the EXCLUDE env var is appended and gets passed as a single argument to imapsync even if it contains whitespace?

https://www.freedesktop.org/software/systemd/man/systemd.service.html#Command%20lines

Reply:
Hum, I clean all empty array in the text area and all whitespace

https://github.com/NethServer/nethserver-mail/blob/master/ui/src/views/Imapsync.vue#L757

Reply:
For instance when I save with empty lines in the textarea or with whitespaces at the end of beginning  of the line I save

Exclude":["kkk","jjj","k"]

Reply:
I ran an experiment with

    Resoconto vendite
    Drafts

They are excluded as expected